cli/cmd/integration_jira.go

```diff
@@ -19,112 +19,63 @@
+	Issue         string
+	Project       string
+	Username      string
+	Token         string
+	Password      string
+	Grouping      string
+	Bidirectional bool
 }
 
-func createJiraCloudAlertChannelIntegration() error {
-	questions := []*survey.Question{
-		{
-			Name:     "name",
-			Prompt:   &survey.Input{Message: "Name: "},
-			Validate: survey.Required,
-		},
-		{
-			Name:     "url",
-			Prompt:   &survey.Input{Message: "Jira URL: "},
-			Validate: survey.Required,
-		},
-		{
-			Name:     "issue",
-			Prompt:   &survey.Input{Message: "Issue Type: "},
-			Validate: survey.Required,
-		},
-		{
-			Name:     "project",
-			Prompt:   &survey.Input{Message: "Project Key: "},
-			Validate: survey.Required,
-		},
-		{
-			Name:     "username",
-			Prompt:   &survey.Input{Message: "Username: "},
-			Validate: survey.Required,
-		},
-		{
-			Name:     "token",
-			Prompt:   &survey.Password{Message: "API Token: "},
-			Validate: survey.Required,
-		},
-	}
-
-	var answers jiraAlertChannelIntegrationSurvey
-	err := survey.Ask(questions, &answers,
-		survey.WithIcons(promptIconsFunc),
-	)
-	if err != nil {
-		return err
-	}
-
-	jira := api.JiraDataV2{
-		ApiToken:  answers.Token,
-		IssueType: answers.Issue,
-		JiraType:  api.JiraCloudAlertType,
-		JiraUrl:   answers.Url,
-		ProjectID: answers.Project,
-		Username:  answers.Username,
-	}
-
-	// ask the user if they would like to configure a Custom Template
-	custom := false
-	err = survey.AskOne(&survey.Confirm{
-		Message: "Configure a Custom Template File?",
-	}, &custom)
-
-	if err != nil {
-		return err
-	}
-
-	if custom {
-		var content string
+func getJiraGroupingOptions() []string {
+	options := make([]string, 0, len(api.JiraIssueGroupingsSurvey))
 
-		err = survey.AskOne(&survey.Editor{
-			Message:  "Provide the Custom Template File in JSON format",
-			FileName: "*.json",
-		}, &content)
-
-		if err != nil {
-			return err
-		}
-
-		jira.EncodeCustomTemplateFile(content)
+	for option := range api.JiraIssueGroupingsSurvey {
+		options = append(options, option)
 	}
 
-	jiraCloudAlertChan := api.NewAlertChannel(answers.Name, api.JiraAlertChannelType, jira)
+	sort.SliceStable(options, func(i, j int) bool {
+		return api.JiraIssueGroupingsSurvey[options[i]] < api.JiraIssueGroupingsSurvey[options[j]]
+	})
 
-	cli.StartProgress(" Creating integration...")
-	_, err = cli.LwApi.V2.AlertChannels.Create(jiraCloudAlertChan)
-	cli.StopProgress()
-	return err
+	return options
 }
 
-func createJiraServerAlertChannelIntegration() error {
+func createJiraAlertChannelIntegration(jiraType string) error {
 	questions := []*survey.Question{
 		{
 			Name:     "name",
 			Prompt:   &survey.Input{Message: "Name: "},
 			Validate: survey.Required,
 		},
+		{
+			Name: "bidirectional",
+			Prompt: &survey.Confirm{
+				Message: "Would you like a bidirectional integration?",
+				Default: false,
+				// Help:    "See https://docs.lacework.com/onboarding/jira#bidirectional-integration for more detail.",
```

Comment:
Are we going to include the help text?

Reply:
Yes, I had commented this and posed a question to #tech-ally . @afiune confirmed we can include doc links.

Reply:
Fixed.